### ISSUE TITLE
NumberPicker's custom focus highlight should be styled the same as the default focus highlight

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#338]
Currently the highlight around `NumberPicker` is a bit thicker than the default. We should solve this in the same way as https://github.com/phetsims/scenery/issues/676. @jessegreenberg do you agree?

**Comment by jessegreenberg** [comment]
Agreed,  https://github.com/phetsims/scenery/issues/676 would help keep consistent styling for thickness (Thought the rounded corners of highlight was a custom request for NumberPicker from https://github.com/phetsims/gravity-force-lab-basics/issues/15).

**Comment by zepumph** [comment]
I found that the problem with NumberPicker's focusHighlight is that currently FocusHighlightPath doesn't inherit scale from the node that it is focusing. In GFLB, the NumberPicker is scaled to 1.5x, so the focushighlight seems thicker. So the following code has twice as thick lineWidth:
```js
    this.focusHighlight = new FocusHighlightPath( /* shape with radii */ )
```

A gross solution is to manually calculate the linewidth from FocusHighlightPath's static methods and then pass that in like so:
```js
    this.focusHighlight = new FocusHighlightPath( /* shape with radii */ , 
       {
        innerLineWidth: FocusHighlightPath.getInnerLineWidthFromNode( this ),
        outerLineWidth: FocusHighlightPath.getOuterLineWidthFromNode( this )
      }
);
```
That way the lineWidth is based on the Node that the focusHighlight is surrounding. Another option may be to pass the scale into the focushighlighPath, but that doesn't work right off the deck. I think that's because we are accounting for that already in the `getMagnituge` method below.
```js
    getWidthMagnitudeFromTransform: function( node ) {
      return node.transform.transformDelta2( Vector2.X_UNIT ).magnitude();
    }
```


I'm not sure the best way to go about doing this. It would be nice if the FocusHighlightPath had a reference to the node that it is trying to focus around. Then it would be able to make better decisions about sizing and line width, rather than the client needing to do that.

**Comment by zepumph** [comment]
In the above commits, I made two fixes. One was kinda totally separate from this issue because NumberPicker doesn't use FocusHighlightFromNode

**FocusHighlightFromNode now will adjust the line width based on the the node it is highlighting.**

Then I "fixed" this issue proper with a temporary workaround where I am manually setting the lineWidth of the focusHighlight based on the scale of the node. This is a problem because FocusHighlightPath doesn't have any context about the transform of the node it is highlighting, but it should for this case. Marking for a11y meeting so that we can discuss the best solution.

I think that ideally the FocusHighlightPath should be able to know how to adjust the lineWidth based on the transform of the node it is wrapping. This may be trying to get too clever though. Do we need another focus highlight type?

**Comment by zepumph** [comment]
Thanks @jessegreenberg and @jonathanolson for assisting me in this problem, you both were very helpful in getting juices flowing in ze brain! I ended up going with a strategy that was based more on the node that the focusHighlight is highlighting (sorry if that is confusing). Rather than scaling based on the transform of the focusHighlight, I'm doing so based on the node. It is working pretty well, but I want to talk strategy before fully signing off on it. @jessegreenberg let's talk about this together sometime soon, like a pair review. I think it will be quicker that you diving into it, but if that is what you're feeling go crazy!

**Comment by zepumph** [comment]
@jessegreenberg please review. Anything else for this issue?

**Comment by pixelzoom** [comment]
Assigned to @jessegreenberg for review for 39 months. Raising priority to high. @jessegreenberg when can you get to this?

**Comment by jessegreenberg** [comment]
Nothing else for this issue. The result of this issue was FocusHighlightPath.getDilationCoefficient which has been working well since 2017.  It looks like comments from https://github.com/phetsims/scenery-phet/issues/338#issuecomment-341850781 were discussed in a meeting on Nov 9 2017.  I reviewed the above commits again and code in FocusHighlightPath, things look good. Closing.
